packages/connectors/react/src/inject.jsx

```diff
@@ -45,14 +45,13 @@ export default injectFunction => WrappedComponent => class extends Component {
       )
       return
     }
-
-    // subscribe
-    this.unsubscribe = store.subscribe(() => {
-      this.inject()
-    })
+    this.store = store
 
     // run in once
     this.inject()
+
+    // subscribe
+    this.unsubscribe = store.subscribe(() => { this.inject() })
```

Comment:
You can write it : 
```js
this.unsubscribe = store.subscribe(this.inject)
```

Reply:
I did it on purpose because I don't know if redux API will pass parameter in future.
And if they do, then inject will not work as attended.